contracts/snowglobes/traderJoe/snowglobe-joe-avax-fief.sol

```diff
@@ -0,0 +1,136 @@
+// https://github.com/iearn-finance/vaults/blob/master/contracts/vaults/yVault.sol
+
+pragma solidity ^0.6.7;
+
+import "../../interfaces/controller.sol";
+
+import "../../lib/erc20.sol";
+import "../../lib/safe-math.sol";
+
+contract SnowGlobeJoeAvaxFief is ERC20 {
```

Comment:
I looked at two other Snowglobes in this trader joe folder and they are all the same except for the filenames. I see no need to duplicate code like this. We should instead create a single snowglobe contract that can be redeployed, if it's necessary to know the name of the pair and dex then add them as additional constructor params. This will also make the codebase less overwhelming for future dev.